academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,116 @@
+description: Step by step tutorial to link private paper data with MAKES paper entities
+ms.topic: tutorial
+ms.date: 10/15/2020
+---
+
+# Entity Linking
+
+his tutorial illustrates how to link private publication records with their corresponding MAKES paper entities, enabling custom entity metadata to be generated.
+
+## Prerequisites
+
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
+- [Powershell 7](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7&preserve-view=true)
+- [Sample private library publication records](samplePrivateLibraryData.json)
+- [Sample entity linking Powershell script](linkPrivateLibraryData.ps1)
+
+## Determine entity linking strategy
+
+Once we know what entity we want to link our data against, we can determine what's the appropriate linking strategy. In abstract, we'll leverage MAKES' [natural language processing](concepts-queries.md) capability via Interpret API to link entities.
+
+MAKES' default grammar supports matching natural language queries against various paper attributes (i.e. title, author names, fields of study, title/abstract terms, etc.), resulting in a query expression which can be used to retrieve corresponding paper entities. You can leverage the Interpret test page to explore what type of query makes the most sense for your scenario.
+
+In this tutorial, our sample library data includes a title so we can leverage MAKES' paper title search capability to link our data. We achieve this by sending Interpret requests to a MAKES instance with the query being **title: [library paper title]** as the following:
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_request":::
+
+You can also link different types of entities together with the default grammar. For example, you can link author data with MAKES papers by sending Interpret requests with the following query format "**[author name] [affiliation name]**".
+
+You can also build a custom grammar to better fit your linking scenario. See [How to define a custom grammar](how-to-grammar.md) and [How to compile and test a custom grammar](how-to-grammar-build.md) for more detail
+
+## Set appropriate confidence score for linking
+
+Lastly, we'd have to determine an acceptable confidence score to link entities. Interpret API returns a log probability associated with each interpretation. This log probability can be seen as a "confidence score" for the given interpretation. Since the probability for an interpretation can range from 0 to 1, the log probability for an interpretation can be anywhere from negative infinity to 0.
+
+The interpretation log probability also included the log probability of the top matching entity. To get the pure interpretation log probability without considering the interpreted entity, we must subtract the entity log probability from the interpretation log probability.
+
```

Comment:
"This interpretation log probability value represents the sum of two different probabilities. The first is the probability assigned by the natural language grammar that the Interpret API uses, and represents the probability that an interpretation is correct based on rules defined in the grammar. The second is the probability associated with the **top entity matching the interpretation**.

For entity linking we want to isolate the **grammar log probability** as it reflects the quality of the match. This is due to how the grammar penalizes certain types of matching, e.g. favoring complete title matches over individual keywords, heavily penalizing terms that can't me matched with an attribute, etc. 

To isolate the grammar log probability we subtract the top matching entities log probability: ..."